### DISCUSSION TITLE
Add cards for enclosures and other facilities

### DISCUSSION THREAD

**Discussion by pjeweb** [alveusgg/extension#20] [Ideas]
![Card mockup for enclosures and facilities](https://user-images.githubusercontent.com/684458/213874332-f8ae8f6b-b7c6-4cab-963d-513afa3833d3.png)

- This would require an additional button to change the list or some other UI
- Each Card could have a Popup similar to those of the ambassadors with:
   - Images
   - Info about Cost and Sponsors
   - List of housed ambassadors (linking to their popup)
   - Links to timestamped video of the alveus tour videos
- You could also back-link from the ambassadors to the enclosure Popup and/or the related ambassadors in the same enclosure/facility

I started to collect some JSON with infos:
https://alveusgg.github.io/data/enclosures.json
https://alveusgg.github.io/data/facilities.json

**Comment by abdullahmorrison** [comment]
I'm open to it, but I don't want to clutter the user with too much info. So, I want to make sure that the only info they need is one that helps them in animal conservation and I don't think knowing about the enclosure is impactful enough.

**Reply by abdullahmorrison** [reply]
But, if designed in an interesting way, it could be doable. Like it would be cool if we could click on a button that opens up a map of alveus and that would have the enclosures to click on and learn more about them and what ambassadors are in them. That would require a crap ton of work tho. Don't think it's worth.
